executor/command.go

```diff
@@ -33,10 +34,13 @@ func (e *Executor) createCmd(paths requestPaths, req execute.Request) *exec.Cmd
 	// Separate runtime arguments from the function arguments.
 	args = append(args, "--")
 
-	// Function arguments.
+	// Append just ordinal Function arguments first
+	args = append(args, req.Arguments...)
+
+	// Append Function parameters with values
 	for _, param := range req.Parameters {
 		if param.Value != "" {
-			args = append(args, param.Value)
+			args = append(args, param.Name + "=" + param.Value)
```

Comment:
What's the difference between arguments and parameters? I don't think this brings anything compared to the previous version, functionality-wise. Speaking from the POV of just _I am executing a binary and giving it some arguments_.

The same thing can be done with the previous version by using:

```json
{
	"parameters": [
		{ "value": "key1=val1" },
		{ "value": "key2=val2" 	}
	]
}
```

I don't think executables typically use `key=val` syntax for CLI flags (with no dashes or anything). Not enough that we have such first-class support for that syntax.

Plus, with the previous version we can support all variants:

a) key-value pairs
```go
	 []execute.Parameter{
		{Value: "key1=val1"},
		{Value: "key2=val2"},
		{Value: "key3=val3"},
	}
```

b) Typical UNIX flags:

```go
	 []execute.Parameter{
		{Value: "--file"},
		{Value: "/tmp/whatever.txt"},
		{Value: "--size"},
		{Value: "64MB"},
	}
```

c) Windows whatevers:

```go
	[]execute.Parameter{
		{Value: "dir"},
		{Value: "/s"},
		{Value: "/q"},
	}
```

If anything, we can just remove `Parameters` completely and just use a string array.